core/src/avm2/globals/flash/desktop/InteractiveIcon.as

```diff
@@ -0,0 +1,27 @@
+// The initial version of this file was autogenerated from the official AS3 reference at
+// https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/desktop/InteractiveIcon.html
+// by https://github.com/golfinq/ActionScript_Event_Builder
+// It won't be regenerated in the future, so feel free to edit and/or fix
+
+package flash.desktop
+{
+  [API("661")]
+  public class InteractiveIcon extends Icon
+  {
+    // The current display height of the icon in pixels.
+    private var _height:int;
+
+    // The current display width of the icon in pixels.
+    private var _width:int;
+
+    public function get height():int
+    {
+      return this._height;
+    }
+
+    public function get width():int
+    {
+      return this._width;
+    }
```

Comment:
Since `_width` and `_height` don't seem to be set (or even be able to set), I don't see the point of them. The getters could just return a constant `0` or `-1` or `undefined`.
Also, a `stub_getter` call would be appropriate here, like in other places, I think.